### DISCUSSION TITLE
How much time take for github sponsers

### DISCUSSION THREAD

**Discussion by DevKeyurkasvala** [community/community#51521] [Other Feature Feedback, Questions, & Ideas] [answered]
I just submit application for become github sponsers.

How much time it will take by staff for approval?

**Comment by CodeMaster7000** [comment, marked as answer]
Hi @DevKeyurkasvala,

It will generally take a few days for GitHub staff to review your application, although this depends on how busy the staff is at the moment. Once your application has been approved, your profile will automatically be live on [GitHub Sponsors](https://github.com/sponsors). 

You can read more about the topic [here](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account).

**Comment by Anmol-Baranwal** [comment]
The approval process for becoming a GitHub sponsor can vary, but typically it takes a few business days to a week for the GitHub staff to review and approve your application.

During the approval process, the GitHub staff will review your application and verify that you meet the requirements to become a sponsor. They may also reach out to you for more information or to clarify any details in your application.

Once your application is approved, you'll receive an email from GitHub with information on how to set up your sponsorship account and start sponsoring developers and projects on the platform.

If you haven't heard back from GitHub after a week or two, you can try reaching out to their support team for an update on the status of your application.

**Reply by Anmol-Baranwal** [reply]
@DevKeyurkasvala 
I wanted to follow up and ask if you have any additional questions or concerns about my previous response on GitHub. If you have no further inquiries, would you kindly consider marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by Harries** [reply]
a week  means  five working days?

**Comment by minodesign** [comment]
hallo Dev ;-)
it may takes some weeks, depending on how they are busy _now_.
I remember I did the subscription very early, a few weeks later receiving the first newsletter about Sponsors, probably in 2020.

It may help you - before to ask for the Sponsor, to upload and pull all your projects (that you may want make public),
and also use Github for some private repo... So that the staff see that you're using a lot the platform.

Then, getting people paying via the Sponsor will be a way harder challenge, but it's the second step.
(If it's not working and you're a 100% dev, try other forms od monetisation also)

**Comment by griffi-gh** [comment]
Years lol

**Comment by mdjakkariya** [comment]
yes it takes few years to get approved, mine still on the way
